chapi/chapiclient.go

```diff
@@ -409,10 +411,10 @@ func (chapiClient *Client) GetDevices() (devices []*model.Device, err error) {
 	_, err = chapiClient.client.DoJSON(&connectivity.Request{Action: "GET", Path: devicesURI, Header: chapiClient.header, Payload: nil, Response: &chapiResp, ResponseError: &chapiResp})
 	if err != nil {
 		if errResp != nil {
-			util.LogError.Print(errResp.Info)
+			log.Error("GetDevices: error response", errResp.Info)
```

Comment:
Should this be the following to align with your other update (directly below).  Otherwise the error is going to append directly after "response".
`log.Error("GetDevices: error response, ", errResp.Info)`

Reply:
actuall logger will add space in this case, but i agree we can keep it consistent.

Reply:
I don't see the above behavior.  Consider this line of code:
```log.Error("GetDevices: error response", "Hello World")```
It produces this in my log file:
```time="2019-08-19T11:54:29-07:00" level=error msg="GetDevices: error responseHello World" file="chapid_windows.go:83"```